docs/support/account.md

```diff
@@ -32,66 +34,56 @@ To manage your security info any time, go to <a href="https://myaccount.microsof
 <hr/></details><br>
 
 <details>
-<summary style="font-size:18px">I have lost the mobile device which I used for MFA authentications. What should I do?</summary>
+<summary style="font-size:18px">I have lost my mobile device that I used for MFA authentications. What should I do?</summary>
 
-**If you are a vendor**:
+**If your TechPass login ID's domain is ```techpass.gov.sg```**:
```

Comment:
For this section, the styling seems different. There's a lot of bold sentences/words that's not consistent with the rest.